.travis.yml

```diff
@@ -27,19 +27,19 @@ matrix:
     - jdk: openjdk8
       env:
         - DESC="install (openjdk8)"
-        - CMD="./build-metafile-gen.sh && mvn install && git diff"
+        - CMD="./build-metafile-gen.sh && ./build-patch-filter.sh && mvn install && git diff HEAD~2 HEAD~1 > target/show.patch  && mvn clean verify && git diff"
 
     # JDK 11 (Long Term Support version)
     - jdk: openjdk11
       env:
         - DESC="install (openjdk11)"
-        - CMD="./build-metafile-gen.sh && mvn install && git diff"
+        - CMD="./build-metafile-gen.sh && ./build-patch-filter.sh && mvn install && git diff HEAD~2 HEAD~1 > target/show.patch  && mvn clean verify && git diff"
```

Comment:
remove `&& git diff HEAD~2 HEAD~1 > target/show.patch  && mvn clean verify`

`install` already to `verify`

Reply:
done